geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -154,6 +163,24 @@ public LocatorClusterManagementService(
     this.operationManager = operationManager;
   }
 
+  @VisibleForTesting
+  DistributedLockService getCmsDlockService() {
```

Comment:
This method should be `synchronized` since cmsDlockService is lazy initialized.

Reply:
good catch.